### ISSUE TITLE
igbinary support

### ISSUE THREAD

**Issue by tvandervorm** [iuscommunity-pkg/php70u-pecl-redis#1]
This is a copy of https://github.com/iuscommunity/wishlist/issues/146 for administation purposes.

The compile flag --enable-redis-igbinary needs to be added to enable igbinary serialization support in redis
(see https://github.com/phpredis/phpredis#installation).

**Comment by carlwgeorge** [comment]
Thanks for pointing this out for us.  We build both the php56u and php71u versions of this package with igbinary support already.  It looks like when I created the original package, I [disabled igbinary support](https://github.com/iuscommunity-pkg/php70u-pecl-redis/commit/8d533b65d1f7df94b7a46fb1c91cf2280a0138bf) because at the time we didn't yet have a php70u-pecl-igbinary package to build against.  We do now, so I've enabled the option.  That will bring parity between our php56u/php70u/php71u packages of the redis module.  [php70u-pecl-redis-3.1.3-2](https://github.com/iuscommunity-pkg/php70u-pecl-redis/releases/tag/3.1.3-2) has this change and is now available in our testing repositories.
